pootle/apps/pootle_fs/management/commands/init_fs_project.py

```diff
@@ -94,11 +93,15 @@ def handle(self, **options):
             project = Project.objects.create(
                 code=code,
                 fullname=name,
-                localfiletype=options['file_type'],
                 treestyle='none',
                 checkstyle=options['checkstyle'],
-                source_language=source_language
-            )
+                source_language=source_language)
+            for filetype in options["filetypes"] or ["po"]:
```

Comment:
Just for the record this is not what I got, but I hit another different issue (https://bugs.python.org/issue16399). My stance is to land the PR as it is.